Plugin.pm

```diff
@@ -1212,7 +1212,11 @@ sub _trackItem {
 		$item->{work} = $track->{work};
 	}
 
-	if ($track->{released_at} && $track->{released_at} > time) {
+	if ( $track->{album} ) {
+		$item->{year} = $track->{album}->{year} || (localtime($track->{album}->{released_at}))[5] + 1900 || 0;
```

Comment:
Haha... I wanted to comment that this would never be "0", as we'd always add 1900 to the result of `released_at`. But then I figured out I'm using the same logic all over the plugin...